aikau/src/main/resources/alfresco/forms/TabbedControls.js

```diff
@@ -106,13 +108,14 @@ define(["alfresco/layout/AlfTabContainer",
        * @return {object[]} An array of the form controls to iterate over.
        */
       getFormLayoutChildren: function alfresco_forms_TabbedControls__getFormLayoutChildren() {
-         var _tabbedFormControls = [];
-         if (this.tabContainerWidget)
-         {
-            array.forEach(this.tabContainerWidget.getChildren(), lang.hitch(this, function(contentPane) {
-               _tabbedFormControls = _tabbedFormControls.concat(contentPane.getChildren());
+         var _tabbedFormControls = new Deferred();
+         when(this.getTabContainerChildren(), lang.hitch(this, function(children) {
+            var promisedControls = [];
+            array.forEach(children, lang.hitch(this, function(child) {
+               promisedControls = promisedControls.concat(child.getChildren());
             }));
-         }
+            _tabbedFormControls.resolve(promisedControls);
+         }));
          return _tabbedFormControls;
```

Comment:
Presumably this won't cause a problem with existing usage of `getFormLayoutChildren` expecting a concrete array, not a promise?